src/common/test/test_common.h

```diff
@@ -16,14 +18,17 @@ extern int usleep(useconds_t usec);
 #endif
 
 /* I/O helper methods to retry binding to sockets. */
-static inline UT_string *bind_ipc_sock_retry(const char *socket_name_format,
-                                             int *fd) {
-  UT_string *socket_name = NULL;
+static inline std::string bind_ipc_sock_retry(const char *socket_name_format,
```

Comment:
@pcmoritz unrelated to this PR, but there's no reason for this to be inline, right?